plugins/flatpak/gs-flatpak.c

```diff
@@ -3418,8 +3420,39 @@ gs_flatpak_app_get_copyable (GsFlatpak *self,
 			     GCancellable *cancellable,
 			     GError **error)
 {
-	if (copyable)
-		*copyable = (gs_flatpak_get_collection_id (self, app) != NULL);
+	if (copyable) {
```

Comment:
Can you just early return here if !copyable and save a whole block of indent? :)